docs/cloud-native-security/kspm.asciidoc

```diff
@@ -13,6 +13,7 @@ This integration supports Amazon EKS and unmanaged Kubernetes clusters. For setu
 [sidebar]
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* KSPM is supported on commercial cloud only. Government cloud is not supported (https://github.com/elastic/enhancements[request support]).
```

Comment:
Maybe we can make this line a bit more specific? Something like: Deploying KSPM on EKS clusters in AWS Government Cloud is not supported.